packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/BlocksInput/BlocksToolbar.tsx

```diff
@@ -609,9 +610,16 @@ interface MoreMenuProps {
   hasHiddenItems: boolean;
   rootRef: React.RefObject<HTMLElement>;
   children: React.ReactNode;
+  triggerVariant?: IconButtonProps['variant'];
 }
 
-const MoreMenu = ({ setLastVisibleIndex, hasHiddenItems, rootRef, children }: MoreMenuProps) => {
+export const MoreMenu = ({
+  setLastVisibleIndex,
+  hasHiddenItems,
+  rootRef,
+  children,
+  triggerVariant = 'tertiary',
+}: MoreMenuProps) => {
```

Comment:
in my opinion this should be moved into your EditorToolbarObserver file. Same for ObservedToolbarComponent and the ObservedComponent type

Reply:
Yes it is in there, I haven't update blocks